bot/server.go

```diff
@@ -130,5 +156,21 @@ func ServeBot(path *string, port *int) {
 	r.HandleFunc("/predict", bot.predictHandler).Methods("POST")
 	r.HandleFunc("/senders/{sender}", bot.detailsHandler).Methods("GET")
 
-	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%v", *port), r))
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", *port), r))
+}
+
+func writeAnswer(w http.ResponseWriter, answers []query.Answer) {
```

Comment:
This was pulled out of the `Channel` clients so the channel implementation can be a little more generic.